packages/calcite-components/src/components/combobox/combobox.scss

```diff
@@ -243,7 +252,10 @@ calcite-chip {
 }
 
 .chip--active {
```

Comment:
Q: @ashetland chip currently do not support this pattern of changing background when active, is this intentional?

Reply:
Yes this is intentional. However, this behavior was recently add for `selectable` chips in PR #11538. Since the chips in Combobox are not themselves selectable, I don't think we should add `--calcite-combobox-chip-background-color-active`.

Reply:
discovered that `chip--active` class NO effect on the chip `backgroundColor`. Setting `backgroundColor` of the chip at `:host` level wont work with chip offering a token to customize it. 

This was working till [2.12.0 version](https://codepen.io/anvesh-mekala/pen/EaxmxpK) and with chip tokenization [10179](https://github.com/Esri/calcite-design-system/pull/10179), backgroundColor no longer changes. After discussing with @ashetland , removing this class so that we preserve the backgroundColor of chip in combobox.

cc @ashetland , @macandcheese